functions.Rmd

```diff
@@ -57,6 +59,24 @@ add_two <- function(x, y) {
 }
 ```
 
+Note that return statements should always be on their own line because they have important effects on the control flow. See also [inline statements](#inline-statements).
```

Comment:
```suggestion
Return statements should always be on their own line because they have important effects on the control flow. See also [inline statements](#inline-statements).
```